### ISSUE TITLE
(Was: Obsoletion notice) GO:0044105 L-xylulose reductase (NAD+) activity - add RHEA x ref

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#21874]
EDITED

The proposal was made to obsolete GO:0044105 L-xylulose reductase (NAD+) activity - but evidence for this activity was found and a RHEA reaction was created. 

Original ticket: 
----

Please provide as much information as you can:

* **GO term ID and Label**
* GO:0044105 L-xylulose reductase (NAD+) activity

* **Reason for deprecation**
~This term was obsoleted by EC and by RHEA.~
There is no evidence that this function exists.

* **"Replace by" term (ID and label)**
If all annotations can safely be moved to that term
NA

* **"Consider" term(s) (ID and label)**
Suggestions for reannotation

NA
* **Are there annotations to this term?**
- How many EXP: 0 

* **Are there mappings and cross references to this term? (InterPro, Keywords; check QuickGO cross-references section)**
EC:1.1.1.10, but this is incorrect, as it describes the NADP reaction, which corresponds to GO:0050038 L-xylulose reductase (NADP+) activity

* **Is this term in a subset? (check the AmiGO page for that term)**
No 

* **Any other information**

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0044105 L-xylulose reductase (NAD+) activity. The reason for obsoletion is that there is no evidence that this activity exists. There are no annotations to this term, other than those coming from the incorrect EC mapping. This term is not in any subsets. 

You can comment on the ticket: https://github.com/geneontology/go-ontology/issues/21874

Thanks, Pascale

**Comment by ukemi** [comment]
PMID: 20654618 Second paragraph of introduction?

**Comment by pgaudet** [comment]
@amorgat @kaxelsen This reaction is missing from RHEA and EC 
L-xylitol + NAD(+) <=> L-xylulose + NADH

Can it be added? If yes, I'll restore the term. 

Thanks, Pascale

**Comment by alanbridge** [comment]
interesting, could be a nice TrEMBL here: 

NADH L-xylulose reductase
https://www.uniprot.org/uniprot/Q70FD1

**Comment by kaxelsen** [comment]
I just identified the same paper, pmid:14736891. It is more convincing than pmid:20654618 where the NADH dependent activity is only seen in crude extract. I will create the reaction.

**Comment by alanbridge** [comment]
Thanks @kaxelsen could you let @marcfeuermann know too?

**Comment by kaxelsen** [comment]
Of course

**Comment by alanbridge** [comment]
thanks, I am having a tea break from ISMB so can approve the Rhea too if needed

**Comment by pgaudet** [comment]
OK great, I'll un-obsolete the term.

**Comment by alanbridge** [comment]
this will be one more GO annotation from the Rhea2GO pipeline too... 

should this reaction 

https://www.rhea-db.org/rhea/48760
L-threitol + NAD(+) = H(+) + L-erythrulose + NADH

be linked to EC 1.1.1.9?

Currently that EC is only linked to 

https://www.rhea-db.org/rhea/20433
NAD(+) + xylitol = D-xylulose + H(+) + NADH

but that EC number says 

Also acts as an L-erythrulose reductase.
https://enzyme.expasy.org/EC/1.1.1.9

**Comment by pgaudet** [comment]
Is there a reason the substrate dont have the D - ie xylitol versus D-xylitol?

**Comment by alanbridge** [comment]
It's symmetrical. Only chiral molecules have optical isomers.

**Comment by alanbridge** [comment]
Hi, 

From @kaxelsen and now approved in Rhea

RHEA:68100 NAD+ + xylitol = H+ + L-xylulose + NADH
EC 1.1.1.15
MetaCyc [RXN-11825]

SHAW DR (1956)
Polyol dehydrogenases. 3. Galactitol dehydrogenase and D-iditol dehydrogenase.
The Biochemical journal 64, 394-405 [PMID:13373783]

Granström TB , Izumori K , Leisola M (2007)
A rare sugar xylitol. Part I: the biochemistry and biosynthesis of xylitol.
Applied microbiology and biotechnology 74, 277-281 [PMID:17216457]

Cheers, Alan

**Comment by kaxelsen** [comment]
Unfortunately, I entered a wrong paper in the entry, and the error was missed by the review process:
Replace pmid:17216457 with 

 Verho R , Putkonen M , Londesborough J , Penttilä M , Richard P (2004)
A novel NADH-linked l-xylulose reductase in the l-arabinose catabolic pathway of yeast.
The Journal of biological chemistry 279, 14746-14751 [14736891]

**Comment by pgaudet** [comment]
@kaxelsen Does the description for UniProt:G3YG17 need to be changed to change the NADPH to NADH ?

**Comment by pgaudet** [comment]
WRT taxon constraints, this activity exists in A. niger (G3YG17, PMID:20654618),  in Ambrosiozyma monospora (PMID:17216457), and in some related species. So, I will add a taxon constraint to Ascomycota. 
Neither of these sequences are in Panther, so I cannot be more precise. 

Thanks, Pascale

**Comment by kaxelsen** [comment]
No, pmid:20654618 shows specifically that LxrA is NADPH dependent. It is when you delete that gene that a NADH dependent activity can be found in crude extracts.

**Comment by marcfeuermann** [comment]
Hello,
I've annotated entry Q70FD1 corresponding to Ambrosiozyma monospora NADH L-xylulose reductase. I've added Rhea:68100.
Moreover, in the same article, it is shown that AXL1 also converts D-ribulose to D-arabinitol (Rhea:17389). I've also added this reaction to the entry and associated GO:0047038 to the protein in Protein2GO. AXL1 can perform both reactions in the 2 directions and kinetic parameters have been characterized for all 4 possible substrates.
Cheers,
Marc.
